superset-frontend/src/views/CRUD/welcome/Welcome.tsx

```diff
@@ -118,6 +132,14 @@ const WelcomeNav = styled.div`
   }
 `;
 
+export const LoadingCards = ({ cover }: LoadingProps) => (
+  <CardContainer showThumbnails={cover} className="loading-cards">
+    {[...new Array(8)].map(() => (
```

Comment:
We could export `8` to a constant with a meaningful name.

Reply:
Is this 8 because of the Recents cards? I was wondering if this could actually import/use the `PAGE_SIZE` constant we already export from `CRUD/utils.tsx` which happens to be 5.